ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -180,124 +143,85 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
   };
 
   return (
-    <LoadingSpinnerOverlay
-      isLoading={
-        props.isCgnInfoLoading ||
-        (props.cgnDetails && cardLoading) ||
-        isLoading(props.unsubscriptionStatus)
+    <BonusCardScreenComponent
+      logoUris={logoUris}
+      name={I18n.t("bonus.cgn.name")}
+      organizationName={I18n.t("bonus.cgn.departmentName")}
+      counters={[]}
```

Comment:
What do you think to make `counters` prop as optional value?

Reply:
I personally never write optional props. They are useful in building blocks components such as uikits.
Vast majority of components those are not primitive ui elements are used in few places with small variations.
Making props not optional helps with code discovery, makes clear what is there and what is not in the call sites.

Reply:
Thank you for your explanation, but I believe there’s value in making the counters prop optional in this context.

While I understand your point about making all props required to ensure clarity and consistency, it’s important to recognize that not all components will always need every prop to function properly. Making counters optional reflects its actual usage: in scenarios where there are no counters to display, we shouldn’t need to pass an empty array just to satisfy a required prop.

Optional props provide flexibility and make the component more adaptable to different use cases. It also improves code readability by reducing unnecessary boilerplate, like passing empty values just to satisfy prop requirements.

Additionally, making props optional does not necessarily hinder code discovery. Clear documentation and well-defined default values can achieve the same level of clarity while keeping the component flexible and easier to use across different contexts.

Let me know what you think.

Reply:
done dfc14b1f4db033770326c791683549b777e4344d